x-pack/plugins/ingest_manager/common/types/models/settings.ts

```diff
@@ -11,6 +11,7 @@ export interface BaseSettings {
   kibana_urls: string[];
   kibana_ca_sha256?: string;
   has_seen_add_data_notice?: boolean;
+  additional_yaml_config?: string;
```

Comment:
I am wondering if we should have it on the `output` saved object instead of settings? as it's applied to the output.

Also maybe we should have one for the kibana settings too?

Reply:
What settings for Kibana?

Reply:
do we want to be able to provide certificate for connecting to Kibana too (it's probably another task)?  the settings we have here are for `agents => ES`

Reply:
@ph @nchaulet thanks for the feedback. I'd like to avoid designing this feature within this PR. I believe it implements the linked issue as described but that said, let's not create a migration pain for adding new features.

Reply:
👍  I still think this should probably be saved on the output saved object instead of the settings one

Reply:
👍 will do. It had a name like es_output_config_yaml at one point, but I didn't think of putting it on a different SO 😀

Reply:
I agree with @jfsiii, the goal of this is to fix the self signed certificate for ES, when you enroll with Kibana you can specify the CA and we will keep it.